.github/workflows/build.yml

```diff
@@ -31,7 +31,7 @@ jobs:
       run: pip install typing
 
     - name: Install build backend
-      run: pip install poetry
+      run: pip install setuptools wheel
```

Comment:
You should not need this, pip installs them on demand.

Reply:
Only after #18, right now we don't build in an isolated environment.